MessengerOnFileForLinux/Common/src/SignalHandling.hpp

```diff
@@ -14,4 +14,13 @@ namespace SignalHandling
                                                                                     SIGCONT,
                                                                                     SIGTERM,
                                                                                     0};
-}
+
+
+namespace NCurses
```

Comment:
a nie lepiej klasa zamiast namespaca? #obiektowo

Reply:
proponuje zrobic taska na to, i przemyśleć co chemy na klasy przerobić a co zostawić. Bo mamy też np fileHandling jako czyste funkjce

Reply:
Można zrobić task czy FileHandling nie powinien przejść mocnego refactoru, ale podtrzymuję, że powinniśmy wywalić te funkcje do klasy

Reply:
ok